mmil.raw.html

```diff
@@ -9221,18 +9221,6 @@
   developed.</TD>
 </TR>
 
-<TR>
-  <TD>eflt</TD>
-  <TD>~ efltim</TD>
-  <TD>The set.mm proof of the converse relies on ltord1</TD>
-</TR>
-
-<TR>
-  <TD>efle</TD>
-  <TD>~ efler</TD>
-  <TD>The set.mm proof of the converse relies on eflt</TD>
-</TR>
-
```

Comment:
These theorems were already proven before? Then this is only a clean up of this list?

Reply:
No, `eflt`, `efle`, and `reefiso` were not proved before in iset.mm.  They are added by this pull request.